docs/source/dynamo/troubleshooting.rst

```diff
@@ -620,8 +620,9 @@ can be used in coordination with bucketing techniques to achieve an
 acceptable number of recompilations for some dynamic models.
 
 .. code-block:: python
-
-   prof = dynamo.utils.CompilationProfiler()
+   from torch._dynamo.utils import CompileProfiler
+   
+   prof = CompileProfiler()
    @dynamo.optimize(prof)
```

Comment:
If we're on the topic of modernizing the doc more, it would be good to explain how to apply this to a torch.compile call

Reply:
im gonna keep that separate. i can do another PR to hit all the dynamo.optimize usages